### PR TITLE
fix: php array types

### LINKED ISSUE #619: PHP Syntax Error with Array Type Hints
### Issue Summary
The type hints in multiple files use `string[]` instead of `array`.

### Steps to Reproduce
Run `php -l` on all PHP files.

### Example Type Hint (`src/Twilio/Rest/Api/V2010/Account/Call/FeedbackOptions.php`)
```php
public static function create(string[] $issue = Values::NONE): CreateFeedbackOptions {
```

### Exception/Log
```
PHP Parse error:  syntax error, unexpected '[', expecting variable (T_VARIABLE) in vendor/twilio/sdk/src/Twilio/Rest/Api/V2010/Account/Call/FeedbackOptions.php on line 20
PHP Parse error:  syntax error, unexpected '[', expecting variable (T_VARIABLE) in vendor/twilio/sdk/src/Twilio/Rest/Api/V2010/Account/Call/RecordingOptions.php on line 29
PHP Parse error:  syntax error, unexpected '[', expecting variable (T_VARIABLE) in vendor/twilio/sdk/src/Twilio/Rest/Api/V2010/Account/CallOptions.php on line 73
PHP Parse error:  syntax error, unexpected '[', expecting variable (T_VARIABLE) in vendor/twilio/sdk/src/Twilio/Rest/Api/V2010/Account/Conference/ParticipantOptions.php on line 106
PHP Parse error:  syntax error, unexpected '[', expecting variable (T_VARIABLE) in vendor/twilio/sdk/src/Twilio/Rest/Api/V2010/Account/ConnectAppOptions.php on line 32
PHP Parse error:  syntax error, unexpected '[', expecting variable (T_VARIABLE) in vendor/twilio/sdk/src/Twilio/Rest/Api/V2010/Account/MessageOptions.php on line 41
PHP Parse error:  syntax error, unexpected '[', expecting variable (T_VARIABLE) in vendor/twilio/sdk/src/Twilio/Rest/Chat/V1/Service/Channel/InviteOptions.php on line 28
PHP Parse error:  syntax error, unexpected '[', expecting variable (T_VARIABLE) in vendor/twilio/sdk/src/Twilio/Rest/Chat/V1/Service/Channel/MemberOptions.php on line 28
PHP Parse error:  syntax error, unexpected '[', expecting variable (T_VARIABLE) in vendor/twilio/sdk/src/Twilio/Rest/Chat/V1/Service/ChannelOptions.php on line 33
PHP Parse error:  syntax error, unexpected '[', expecting variable (T_VARIABLE) in vendor/twilio/sdk/src/Twilio/Rest/Chat/V1/ServiceOptions.php on line 167
PHP Parse error:  syntax error, unexpected '[', expecting variable (T_VARIABLE) in vendor/twilio/sdk/src/Twilio/Rest/Chat/V2/Service/BindingOptions.php on line 22
PHP Parse error:  syntax error, unexpected '[', expecting variable (T_VARIABLE) in vendor/twilio/sdk/src/Twilio/Rest/Chat/V2/Service/Channel/InviteOptions.php on line 28
PHP Parse error:  syntax error, unexpected '[', expecting variable (T_VARIABLE) in vendor/twilio/sdk/src/Twilio/Rest/Chat/V2/Service/Channel/MemberOptions.php on line 40
PHP Parse error:  syntax error, unexpected '[', expecting variable (T_VARIABLE) in vendor/twilio/sdk/src/Twilio/Rest/Chat/V2/Service/Channel/WebhookOptions.php on line 31
PHP Parse error:  syntax error, unexpected '[', expecting variable (T_VARIABLE) in vendor/twilio/sdk/src/Twilio/Rest/Chat/V2/Service/ChannelOptions.php on line 38
PHP Parse error:  syntax error, unexpected '[', expecting variable (T_VARIABLE) in vendor/twilio/sdk/src/Twilio/Rest/Chat/V2/Service/User/UserBindingOptions.php on line 21
PHP Parse error:  syntax error, unexpected '[', expecting variable (T_VARIABLE) in vendor/twilio/sdk/src/Twilio/Rest/Chat/V2/ServiceOptions.php on line 100
PHP Parse error:  syntax error, unexpected '[', expecting variable (T_VARIABLE) in vendor/twilio/sdk/src/Twilio/Rest/Conversations/V1/Conversation/WebhookOptions.php on line 34
PHP Parse error:  syntax error, unexpected '[', expecting variable (T_VARIABLE) in vendor/twilio/sdk/src/Twilio/Rest/Conversations/V1/WebhookOptions.php on line 31
PHP Parse error:  syntax error, unexpected '[', expecting variable (T_VARIABLE) in vendor/twilio/sdk/src/Twilio/Rest/IpMessaging/V1/Service/Channel/InviteOptions.php on line 28
PHP Parse error:  syntax error, unexpected '[', expecting variable (T_VARIABLE) in vendor/twilio/sdk/src/Twilio/Rest/IpMessaging/V1/Service/Channel/MemberOptions.php on line 28
PHP Parse error:  syntax error, unexpected '[', expecting variable (T_VARIABLE) in vendor/twilio/sdk/src/Twilio/Rest/IpMessaging/V1/Service/ChannelOptions.php on line 33
PHP Parse error:  syntax error, unexpected '[', expecting variable (T_VARIABLE) in vendor/twilio/sdk/src/Twilio/Rest/IpMessaging/V1/ServiceOptions.php on line 167
PHP Parse error:  syntax error, unexpected '[', expecting variable (T_VARIABLE) in vendor/twilio/sdk/src/Twilio/Rest/IpMessaging/V2/Service/BindingOptions.php on line 22
PHP Parse error:  syntax error, unexpected '[', expecting variable (T_VARIABLE) in vendor/twilio/sdk/src/Twilio/Rest/IpMessaging/V2/Service/Channel/InviteOptions.php on line 28
PHP Parse error:  syntax error, unexpected '[', expecting variable (T_VARIABLE) in vendor/twilio/sdk/src/Twilio/Rest/IpMessaging/V2/Service/Channel/MemberOptions.php on line 40
PHP Parse error:  syntax error, unexpected '[', expecting variable (T_VARIABLE) in vendor/twilio/sdk/src/Twilio/Rest/IpMessaging/V2/Service/Channel/WebhookOptions.php on line 31
PHP Parse error:  syntax error, unexpected '[', expecting variable (T_VARIABLE) in vendor/twilio/sdk/src/Twilio/Rest/IpMessaging/V2/Service/ChannelOptions.php on line 38
PHP Parse error:  syntax error, unexpected '[', expecting variable (T_VARIABLE) in vendor/twilio/sdk/src/Twilio/Rest/IpMessaging/V2/Service/User/UserBindingOptions.php on line 21
PHP Parse error:  syntax error, unexpected '[', expecting variable (T_VARIABLE) in vendor/twilio/sdk/src/Twilio/Rest/IpMessaging/V2/ServiceOptions.php on line 100
PHP Parse error:  syntax error, unexpected '[', expecting variable (T_VARIABLE) in vendor/twilio/sdk/src/Twilio/Rest/Lookups/V1/PhoneNumberOptions.php on line 24
PHP Parse error:  syntax error, unexpected '[', expecting variable (T_VARIABLE) in vendor/twilio/sdk/src/Twilio/Rest/Notify/V1/Service/BindingOptions.php on line 28
PHP Parse error:  syntax error, unexpected '[', expecting variable (T_VARIABLE) in vendor/twilio/sdk/src/Twilio/Rest/Notify/V1/Service/NotificationOptions.php on line 45
PHP Parse error:  syntax error, unexpected '[', expecting variable (T_VARIABLE) in vendor/twilio/sdk/src/Twilio/Rest/Preview/HostedNumbers/AuthorizationDocumentOptions.php on line 30
PHP Parse error:  syntax error, unexpected '[', expecting variable (T_VARIABLE) in vendor/twilio/sdk/src/Twilio/Rest/Preview/HostedNumbers/HostedNumberOrderOptions.php on line 35
PHP Parse error:  syntax error, unexpected '[', expecting variable (T_VARIABLE) in vendor/twilio/sdk/src/Twilio/Rest/Preview/TrustedComms/PhoneCallOptions.php on line 60
PHP Parse error:  syntax error, unexpected '[', expecting variable (T_VARIABLE) in vendor/twilio/sdk/src/Twilio/Rest/Preview/Wireless/RatePlanOptions.php on line 32
PHP Parse error:  syntax error, unexpected '[', expecting variable (T_VARIABLE) in vendor/twilio/sdk/src/Twilio/Rest/Proxy/V1/Service/Session/Participant/MessageInteractionOptions.php on line 24
PHP Parse error:  syntax error, unexpected '[', expecting variable (T_VARIABLE) in vendor/twilio/sdk/src/Twilio/Rest/Proxy/V1/Service/SessionOptions.php on line 30
PHP Parse error:  syntax error, unexpected '[', expecting variable (T_VARIABLE) in vendor/twilio/sdk/src/Twilio/Rest/Serverless/V1/Service/BuildOptions.php on line 28
PHP Parse error:  syntax error, unexpected '[', expecting variable (T_VARIABLE) in vendor/twilio/sdk/src/Twilio/Rest/Taskrouter/V1/Workspace/Task/ReservationOptions.php on line 131
PHP Parse error:  syntax error, unexpected '[', expecting variable (T_VARIABLE) in vendor/twilio/sdk/src/Twilio/Rest/Taskrouter/V1/Workspace/TaskOptions.php on line 47
PHP Parse error:  syntax error, unexpected '[', expecting variable (T_VARIABLE) in vendor/twilio/sdk/src/Twilio/Rest/Taskrouter/V1/Workspace/Worker/ReservationOptions.php on line 129
PHP Parse error:  syntax error, unexpected '[', expecting variable (T_VARIABLE) in vendor/twilio/sdk/src/Twilio/Rest/Video/V1/CompositionHookOptions.php on line 56
PHP Parse error:  syntax error, unexpected '[', expecting variable (T_VARIABLE) in vendor/twilio/sdk/src/Twilio/Rest/Video/V1/CompositionOptions.php on line 52
PHP Parse error:  syntax error, unexpected '[', expecting variable (T_VARIABLE) in vendor/twilio/sdk/src/Twilio/Rest/Video/V1/RecordingOptions.php on line 29
PHP Parse error:  syntax error, unexpected '[', expecting variable (T_VARIABLE) in vendor/twilio/sdk/src/Twilio/Rest/Video/V1/RoomOptions.php on line 34
PHP Parse error:  syntax error, unexpected '[', expecting variable (T_VARIABLE) in vendor/twilio/sdk/src/Twilio/Rest/Wireless/V1/RatePlanOptions.php on line 44
```

### Technical details:
* twilio-php version: 6.1.0
* php version: 7.3.10

### PR DESCRIPTION
**Pull request by eshanholtz**
Fixes #619 

No syntax errors detected on any files when running with `php -l`

### CODE REVIEW

**Comment by childish-sambino** [comment]
Think comment docs shouldn't be changed: https://docs.phpdoc.org/latest/guides/types.html#arrays

**Comment by childish-sambino** [comment]
Should we add `php -l` to the unit/travis tests?

**Comment by eshanholtz** [comment]
Not a bad idea. Just have to use a short script to iterate over all the files since `php -l` takes in a single file as an argument. Should I include it as part of this PR?

**Comment by childish-sambino** [comment]
Yeah, why not.

**Comment by ABGEO07** [comment]
@eshanholtz The `Values::NONE` constant is a string ([see](https://github.com/twilio/twilio-php/blob/master/src/Twilio/Values.php#L8)), so you can't assign it to an array. You should use `[]` instead.

File: src/Twilio/Values.php [hunk 1]
```diff
@@ -6,6 +6,7 @@
 
 class Values implements \ArrayAccess {
     public const NONE = 'Twilio\\Values\\NONE';
+    public const ARRAY_NONE = [];
```
Comment (childish-sambino) [inline src/Twilio/Values.php]: This should be `ARRAY_NONE = [NONE]` to align with how `NONE` is defined.

**Pull request merged**